exercises/I-array-properties/exercise.js

```diff
@@ -6,7 +6,9 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  if (arr.length) {
```

Comment:
I think this might be "if(arr.length ===0)" or you may only use "function isEmpty(arr) {return arr.length === 0; }